openvdb_cmd/vdb_tool/CMakeLists.txt

```diff
@@ -0,0 +1,159 @@
+    target_link_libraries(vdb_tool_common INTERFACE blosc)
+  endif()
+endif()
+
+if(USE_PNG)
+  add_compile_definitions("VDB_TOOL_USE_PNG")
+  if(WIN32)
+    find_package(libpng CONFIG REQUIRED)
+  else()
+    find_package(PNG REQUIRED)
+  endif()
+  target_link_libraries(vdb_tool_common INTERFACE png)
+endif()
+
+if(USE_JPG)
+  add_compile_definitions("VDB_TOOL_USE_JPG")
+  find_package(JPEG REQUIRED)
+  target_link_libraries(vdb_tool_common INTERFACE ${JPEG_LIBRARIES})
+  target_include_directories(vdb_tool_common INTERFACE ${JPEG_INCLUDE_DIR})
+endif()
+
+if(USE_EXR)
+  add_compile_definitions("VDB_TOOL_USE_EXR")
+  find_package(OpenEXR REQUIRED)
+  target_link_libraries(vdb_tool_common INTERFACE OpenEXR::IlmImf)
+endif()
+
+if(USE_ABC)
+  add_compile_definitions("VDB_TOOL_USE_ABC")
+  find_package(Alembic CONFIG REQUIRED)
+  target_link_libraries(vdb_tool_common INTERFACE Alembic::Alembic)
+endif()
+
+if(WIN32 AND (USE_ALL OR (USE_ABC AND USE_EXR)))
+  message(WARNING
+    " The OpenEXR and Alembic VCPKG packages are using conflicting Imath versions.\n"
+    " Disable one, if you encounter unresolved external symbols")
+endif()
+
+# Compiler flags
+
+# GCC flags
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  add_compile_options(-Wno-invalid-offsetof -pthread -lpthread)
+  add_compile_options("$<$<CONFIG:DEBUG>:-O1>")
+endif()
+
+
+# MSVC flags
+# Increase the number of sections that an object file can contain
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/bigobj>")
+# Excludes APIs such as Cryptography, DDE, RPC, Shell, and Windows Sockets
+add_compile_definitions("$<$<CXX_COMPILER_ID:MSVC>:WIN32_LEAN_AND_MEAN>")
+# Disable non-secure CRT library function warnings
+# https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/
+#     compiler-warning-level-3-c4996?view=vs-2019#unsafe-crt-library-functions
+add_compile_definitions("$<$<CXX_COMPILER_ID:MSVC>:_CRT_SECURE_NO_WARNINGS>")
+# Disable POSIX function name warnings
+# https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/
+#     compiler-warning-level-3-c4996?view=vs-2019#posix-function-names
+add_compile_definitions("$<$<CXX_COMPILER_ID:MSVC>:_CRT_NONSTDC_NO_WARNINGS>")
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  add_compile_options("$<$<CONFIG:RELEASE>:/Oi>")
+
+  message(STATUS "Suppressing some noisy MSVC CXX warnings.")
+endif()
+# Conversion from int64_t to long
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4244>")
+# It's not possible to use STL types in DLL interfaces in a portable and
+# reliable way so disable this warning
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4251>")
+# Conversion from size_t to uLong
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4267>")
+# Non dll-interface class used as base for dll-interface class
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4275>")
+# Truncation from 'int' to 'bool'
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4305>")
```

Comment:
All these options should either be removed, moved to the core build system or only applied to the vdb_tool target. As they are they will influence all build components

Reply:
please add any changes to the CMakeLists/txt - it was originally written by someone else (Greg Klar) and I have to admit I still don't know how the cmake system works :)

Reply:
Would you be willing to attempt these changes so you can maintain the builds for components you submit?

Reply:
I can initially comment them out

Reply:
did you also mean line 125, 128 and 130 or just 132 and 134?